build.sbt

```diff
@@ -383,43 +406,51 @@ lazy val readme = scalatex.ScalatexReadme(
 lazy val sharedSettings = Def.settings(
   scalaVersion := ScalaVersion,
   crossScalaVersions := ScalaVersions,
-  crossVersion := CrossVersion.binary,
+  crossVersion := {
+    crossVersion.value match {
+      case old @ ScalaJSCrossVersion.binary => old
+      case _ => CrossVersion.binary
+    }
+  },
   version := LibraryVersion,
   organization := "org.scalameta",
   resolvers += Resolver.sonatypeRepo("snapshots"),
   resolvers += Resolver.sonatypeRepo("releases"),
-  addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
-  libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.1" % "test",
-  libraryDependencies += "org.scalacheck" %% "scalacheck" % "1.13.4" % "test",
+  addCompilerPlugin(("org.scalamacros" % "paradise" % "2.1.0").cross(CrossVersion.full)),
+  libraryDependencies += "org.scalatest" %%% "scalatest" % "3.0.1" % "test",
+  libraryDependencies += "org.scalacheck" %%% "scalacheck" % "1.13.5" % "test",
   scalacOptions ++= Seq("-deprecation", "-feature", "-unchecked"),
-  scalacOptions in (Compile, doc) ++= Seq("-skip-packages", ""),
-  scalacOptions in (Compile, doc) ++= Seq("-implicits", "-implicits-hide:."),
-  scalacOptions in (Compile, doc) ++= Seq("-groups"),
+  scalacOptions.in(Compile, doc) ++= Seq("-skip-packages", ""),
+  scalacOptions.in(Compile, doc) ++= Seq("-implicits", "-implicits-hide:."),
+  scalacOptions.in(Compile, doc) ++= Seq("-groups"),
```

Comment:
The official sbt docs also do
```scala
val foo = (project in file("bar").
  settings(...)
```

I'm strongly against infix application with multiple arguments since they can be hard to parse in my head. Regular Term.Apply is easier to read and it scales more nicely for additional arguments/chained calls. For example, the old `) dependsOn (common)` infix usage made it annoying to append another `.addPlugins (ScalaJSPluginb)` line at the end. For uniformity, I refactored all `in` calls to non-infix. Are you OK with keeping this?